src/index.html

```diff
@@ -9,6 +9,63 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home" class="header__link">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+          >
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="logo"
          class="logo"
        >
```

Reply:
I checked it. It's 500. And as i said before it works from Europe but does not work from China. That's a problem. Google is banned here.

Reply:
okay, got it